kbchat/kbchat.go

```diff
@@ -262,8 +271,7 @@ func (a *API) doFetch(apiInput string) (*bufio.Scanner, error) {
 	return output, nil
 }
 
-// SendMessage sends a new text message on the given channel
-func (a *API) SendMessage(channel Channel, body string) error {
+func (a *API) SendMessageWithResponse(channel Channel, body string) (SendResponse, error) {
```

Comment:
I don't think you need to fork these calls, just make all these things return the response. It's ok if this API breaks.

Reply:
Fixed